docs/src/pages/style/color.md

```diff
@@ -25,20 +24,46 @@ They’ve been designed to work harmoniously with each other.
 For instance, you can refer to complementary primary and accent colors (for example 'red 500' & 'purple A200'), like so:
 
 ```js
-import { red, purple } from 'material-ui/colors';
+import purple from 'material-ui/colors/purple';
+import red from 'material-ui/colors/red';
 
 const primary = red[500]; // #F44336
 const accent = purple['A200']; // #E040FB
 const accent2 = purple.A200; // #E040FB (alternative method)
 ```
 
-### Full Palette
-{{"demo": "pages/style/Color.js"}}
+## Color tool
+
+Material Design team has built an awesome palette configuration tool: [https://material.io/color](material.io/color).
+It will help you create, share, and apply color palettes to your UI, as well as measure the accessibility level of any color combination.
```

Comment:
Wait a sec, I already commented on this line...

It doesn't help apply, and why "share"?

"It will help you create a color palette for your UI"